README.md

```diff
@@ -0,0 +1,35 @@
+<p align="center">
+  <img src="assets/repo/meilisearch-logo-light.svg?sanitize=true#gh-light-mode-only">
+  <img src="assets/repo/meilisearch-logo-dark.svg?sanitize=true#gh-dark-mode-only">
+</p>
+
+<h1 align="center">Meilisearch Documentation</h1>
+
+<h4 align="center">
+  <a href="https://www.meilisearch.com">Website</a> |
+  <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
+  <a href="https://blog.meilisearch.com">Blog</a> |
+  <a href="https://www.meilisearch.com/docs">Documentation</a> |
+  <a href="https://www.meilisearch.com/docs/faq">FAQ</a> |
+  <a href="https://discord.meilisearch.com">Discord</a>
+</h4>
+
+
+<a href="https://app.bors.tech/repositories/28374"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
```

Comment:
We're not using bors rn. Should we remove this line?
```suggestion

```

Reply:
Hmmm…I guess you're right. I'll comment it out instead of deleting since we do intend to have it running again soon, AFAIK.